### ISSUE TITLE
Interactive elements should have visible focus style and logical focus order for accessibility

### ISSUE THREAD

**Issue by HaSistrunk** [sarapis/orservices#56]
## Is your feature request related to a problem? Please describe.
To enable access for users that navigate with the keyboard or other assistive tech, there should be visible focus style for interactive elements that are navigated to via keyboard input. The focus order should be predictable and match the visual layout of the site. Currently there are multiple links and buttons without any focus styles, and some styles that only use color to designate focus.

## Describe the solution you'd like
Tab through each page to test visible focus. Usually, it's best to [avoid removing the default browser focus styles](http://www.outlinenone.com/) (by adding `outline: none` to those elements) if you don't replace with custom styles. When using custom styles, include an outline or other visual indication other than color to indicate focus so that color is not the only visual means of conveying information (see [WCAG SC 1.4.1 Use of Color](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)).

## Additional context
These guidelines are specified in WCAG 2.x Success Criteria [2.4.7 AA (Focus Visible)](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#qr-navigation-mechanisms-focus-visible) and [1.4.7 A (Meaningful Sequence)](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#qr-content-structure-separation-sequence)

**Comment by HaSistrunk** [comment]
There are focus styles on most elements, but if you load a page and just hit "tab" repeatedly you can test this (Tab button moves the focus to the next element and Shift-Tab moves it backward).

Elements that still need a focus indicator (just a color change isn't enough because not everyone can see color differences):
- Navigation links in the header
- Category links on the homepage
- Service category links on the services page. These links don't receive focus at all. Example: 
<img width="345" alt="image" src="https://user-images.githubusercontent.com/15254041/189438728-581c37da-86b7-4c1b-b80a-171bda924176.png">
